modules/apps/object/object-test/src/testFunctional/paths/ObjectPortlet.path

```diff
@@ -42,52 +42,52 @@
 	</tr>
 	<tr>
 		<td>DUPLICATED_ENTRY_VALUE</td>
-		<td>(//div[contains(@class,'dnd-tbody')]//*[contains(text(),'${key_entry}')])[2]</td>
+		<td>(//tbody//*[contains(text(),'${key_entry}')])[2]</td>
 		<td></td>
 	</tr>
 	<tr>
 		<td>ENTRY_COLUMN</td>
-		<td>//div[contains(@class,'dnd-th')]//*[contains(text(),'${key_column}')]</td>
+		<td>//th[contains(.,'${key_column}')]</td>
 		<td></td>
 	</tr>
 	<tr>
 		<td>ENTRY_ID</td>
-		<td>//div[contains(@class,'dnd-tbody')]//div[contains(@class,'table-list-title')]</td>
+		<td>//tbody//div[contains(@class,'table-list-title')]</td>
 		<td></td>
 	</tr>
 	<tr>
 		<td>ENTRY_STATUS</td>
-		<td>//*[text()='${key_entry}']/../following-sibling::div//*[text()='${key_status}']</td>
+		<td>//*[text()='${key_entry}']/../following-sibling::td//*[text()='${key_status}']</td>
 		<td></td>
 	</tr>
 	<tr>
 		<td>ENTRY_VALUE</td>
-		<td>//div[contains(@class,'dnd-tbody')]//*[contains(text(),'${key_entry}')]</td>
+		<td>//tbody//*[contains(text(),'${key_entry}')]</td>
 		<td></td>
 	</tr>
 	<tr>
 		<td>ENTRY_VALUE_BY_POSITION</td>
-		<td>//div[contains(@class,'dnd-tbody')]//div[contains(@class, 'dnd-tr')][${key_order}]//*[contains(text(),'${key_entry}')]</td>
+		<td>//tbody//tr[${key_order}]//*[contains(text(),'${key_entry}')]</td>
 		<td></td>
 	</tr>
 	<tr>
 		<td>ENTRY_VALUE_WITH_ELLIPSIS</td>
-		<td>//div[contains(@class,'dnd-tbody')]//span[contains(@class,'text-truncate') and contains(text(),'${key_entry}')]</td>
+		<td>//tbody//span[contains(@class,'text-truncate') and contains(text(),'${key_entry}')]</td>
 		<td></td>
 	</tr>
 	<tr>
 		<td>ENTRY_VALUE_SPECIFIC_ORDER</td>
-		<td>//div[${resultOrder}]//div[contains(@class,'dnd-tbody')]//*[contains(text(),'${key_entry}')]</td>
+		<td>//div[${resultOrder}]//tbody//*[contains(text(),'${key_entry}')]</td>
 		<td></td>
 	</tr>
 	<tr>
 		<td>ENTRY_KEBAB_MENU</td>
-		<td>//div[contains(@class,'dnd-tbody')]//*[contains(text(),'${key_entry}')]/parent::div/following-sibling::div//button | //div[contains(@class, 'dnd-td') and contains(., '${key_entry}')]/following-sibling::div//button</td>
+		<td>//tbody//*[contains(text(),'${key_entry}')]/parent::td/following-sibling::td//button | //td[contains(., '${key_entry}')]/following-sibling::td//button</td> certo
```

Comment:
Hey @larissacribeiro, should that term “certo” be there?

Reply:
sorry @igor-franca, my bad, these are remains of comments I was doing so I didn't loose track of what I had already corrected! same to the bellow comment 😖 Thank you!